### ISSUE TITLE
Broken triggers in training levels

### ISSUE THREAD

**Issue by vvs-** [opentomb/OpenTomb#232]
Some triggers in *Lara's Home/House* in TR1/3 either doesn't work or work incorrectly.

**Comment by Lwmte** [comment]
In TR1, it seems Core programmers made a hardcoded override for each CD trigger. We need to write trigger override script in the same manner - I opt for exporting it into flipeffects (as it was done in TR2).

And, for TR2 and TR3, of course, there's a bunch of things to be implemented - like assault course clock and race timer.

**Comment by vvs-** [comment]
I meant that in TR1 triggers in the swimming pool activate at a wrong times and the trigger "now press it again and quickly press one of the directions and I jump that way" isn't working. In TR3 triggers for individual exercises inside the home gym are not working.

**Comment by Lwmte** [comment]
Yes, exactly. If you look at TR1 triggers in Lara's Home, you will see that they all are CD triggers. However, for each trigger there were additional conditions. Most likely, they hardcoded special routines for CD triggers, when Lara's Home is loaded.

As for TR3, it's much easier, as gym triggers are flipeffects, not CD triggers. So I just have to write flipeffect for each of them.
